packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -81,6 +81,25 @@ export class MirrorNodeClient {
     private static CONTRACT_RESULT_LOGS_PROPERTY = 'logs';
     private static CONTRACT_STATE_PROPERTY = 'state';
 
+    static acceptedErrorStatusesResponsePerRequestPathMap: Map<string, Array<number>> = new Map([
+        [MirrorNodeClient.GET_ACCOUNTS_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BALANCE_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BLOCK_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BLOCKS_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_CONTRACT_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_CONTRACT_RESULTS_BY_ADDRESS_ENDPOINT, [206, 400, 404]],
+        [MirrorNodeClient.GET_CONTRACT_RESULTS_DETAILS_BY_CONTRACT_ID_ENDPOINT, [400]],
+        [MirrorNodeClient.GET_CONTRACT_RESULT_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_CONTRACT_RESULT_LOGS_ENDPOINT, [400, 404]],
```

Comment:
How are you coming up with these codes? You should be using the [OpenAPI spec file](https://github.com/hashgraph/hedera-mirror-node/blob/main/hedera-mirror-rest/api/v1/openapi.yml). Bonus points if you can replace this map with a dynamic call to the openapi yaml from the corresponding mirror node. We expose this yaml directly.

Reply:
Right now I am not improving on the reality of the mirror node, I think that is ideal, to be honest, but right now am focusing on just refactoring the existent code and those codes were already there as acceptedErrorCodes for these endpoints, in the meaning that we will not retry if we get any of those errors. 
Due to how it is coded, fetching it from the yaml is not feasible at the time, since these `Constants` are being used to replace the `{Placeholders}` and then use them as path for the requests, in that case I need them to match exactly in the client in order to verify against this list. so for the time being I think they will stay hardcoded.

Reply:
We've discussed the creating a client from the Mirror Node OpenAPI spec a few times. 
Let's open a ticket to address this in a future sprint.
Right thing to do.